src/netaddress.cpp

```diff
@@ -916,25 +911,17 @@ std::vector<unsigned char> CService::GetKey() const
     return key;
 }
 
-std::string CService::ToStringPort() const
+std::string CService::ToStringAddrPort() const
 {
-    return strprintf("%u", port);
-}
+    const auto port_str = strprintf("%u", port);
```

Comment:
There is no copy elision because [there is no copy](https://www.youtube.com/watch?v=uAXtO5dMqEI). The standard mandates that in this case there is no temporary to be copied and that the object should be [constructed directly into `port_str`](ttps://stackoverflow.com/questions/38043319/how-does-guaranteed-copy-elision-work#38043447).